tools/scripts/acr-build.ps1

```diff
@@ -102,10 +108,17 @@ Write-Warning "Using $($Registry).azurecr.io."
 $latestTag = "latest"
 # get and set build information from gitversion, git or version content
 $sourceTag = $env:Version_Prefix
+$prereleaseTag = $env:Version_Prerelease
+if ([string]::IsNullOrEmpty($prereleaseTag))
+{
+    # This can be used on a preview branch *not* targetted for release and maintainence.
+    $prereleaseTag = "-alpha"
```

Comment:
If never used let's not add it.

Reply:
I left it as a placeholder for future. It is not needed for for this PR. Like the comment says, its if/when you spawn a preview branch.

Reply:
Let's remove it then and add it when actually needed, and then properly test it at that point.

Reply:
coming up shortly